libraries/AP_BattMonitor/AP_BattMonitor_FuelFlow.cpp

```diff
@@ -0,0 +1,121 @@
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Common/AP_Common.h>
+#include <AP_Math/AP_Math.h>
+#include "AP_BattMonitor.h"
```

Comment:
This will be included automatically by the AP_BattMonitor_FuelFlow header.

Reply:
fixed